### PR TITLE
[4.x - cakephp/http] - add composer/ca-bundle

### PR DESCRIPTION
**Pull request by saeideng**
required  in https://github.com/cakephp/cakephp/blob/597c1e92cf1c66c17b113d8e8fa3e766cd8ebec1/src/Http/Client/Adapter/Curl.php#L25

### CODE REVIEW

File: src/Http/composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "cakephp/core": "^4.0",
         "cakephp/event": "^4.0",
         "cakephp/utility": "^4.0",
+        "composer/ca-bundle": "^1.2",
```
Comment (garas) [inline src/Http/composer.json]: Sync version constraint with main `composer.json`

Comment (saeideng) [reply src/Http/composer.json]: I can update main file to `^1.2`

Comment (garas) [reply src/Http/composer.json]: Both variants are OK.

Comment (dereuromark) [reply src/Http/composer.json]: Where is the prefer lowest run now on master? With travis this used to be a) visible and b) daily run using cronjobs. Now I cant find it anymore.

Comment (saeideng) [reply src/Http/composer.json]: removed when merging 3.x to 4.x

Comment (dereuromark) [reply src/Http/composer.json]: Thats bad :/

**Pull request merged**